tmt/utils.py

```diff
@@ -3006,3 +3019,259 @@ def wait(
             tick *= tick_increase
 
             continue
+
+
+class ValidateFmfMixin:
+    """
+    Mixin adding validation of an fmf node.
```

Comment:
Can we keep the first line separated by an empty one as it was before? It makes it more distinct as a heading and improves the readability. Similarly below for `keys()`, `items()` and friends.

Reply:
Fixed, probably a typo during copy & paste.